conversation_reconstruction_dataflow/conversation_reconstruction/construct_utils/utils/deltas/algorithms/diff_engine.py

```diff
@@ -0,0 +1,45 @@
+"""
```

Comment:
Looks like this is copy of code from another repo? If so, it needs to live in a (sub)directory called "third_party/" Same for other files. :)

Reply:
I'm currently putting deltas under construct_utils/utils/third_party, would that be ok?